static/js/event/event.ts

```diff
@@ -18,14 +18,49 @@
  * Disaster events
  */
 
+import axios from "axios";
 import React from "react";
 import ReactDOM from "react-dom";
 
-import { Page } from "./page";
+import { EventPage, Page } from "./page";
+import { Property } from "./types";
+
+function parseApiResponse(resp): Array<Property> {
+  // Parses response from triples API for later rendering
+  const parsedResponse = [];
+  for (const [property, connectedNodes] of Object.entries(resp)) {
+    for (const nodes of Object.values(connectedNodes)) {
+      parsedResponse.push({ name: property, values: nodes });
+    }
+  }
+  return parsedResponse;
+}
 
 window.onload = () => {
-  ReactDOM.render(
-    React.createElement(Page),
-    document.getElementById("main-pane")
-  );
+  const dcid = document.getElementById("node").dataset.dcid;
+  const nodeName = document.getElementById("node").dataset.nn;
+  const propertiesPromise = axios
+    .get(`/api/node/triples/out/${dcid}`)
+    .then((resp) => resp.data)
+    .catch(() => {
+      return 0;
+    });
+  propertiesPromise
+    .then((apiResponse) => {
+      const parsedResponse = parseApiResponse(apiResponse);
+      ReactDOM.render(
+        React.createElement(EventPage, {
+          dcid: dcid,
+          name: nodeName,
+          properties: parsedResponse,
+        }),
+        document.getElementById("node")
+      );
+    })
+    .catch(() => {
+      ReactDOM.render(
```

Comment:
I don't think we should have a separate component for when there is an error retrieving triples data. We can just render the EventPage component but pass in an empty array for the properties prop. To inform users that there was a problem, could add an alert like `window.alert("Sorry, there was a problem retrieving data about this disaster")`

Reply:
removed separate component